_posts/2021-09-07-what-javascript-tests-could-learn-from-rspec.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: epic
+title: "What JavaScript Tests Could Learn From RSpec"
+date: 2021-09-10
+categories: [testing, rspec, jest]
+author: steve-hicks
+---
+
+TODO: rel="canonical" to the original on my blog
+
+When I started at Artsy a few years ago, I'd never written a line of Ruby. I feel at home with JavaScript — it's
+been my buddy since I started my career over 20 years ago. I've written enough tests in JavaScript that I sometimes
+feel like I can write them in my sleep (as long as they don't involve async React events 😅).
+
+Most of the code I write at Artsy is still JavaScript, but now I write some Ruby code too, and I've written enough
+RSpec tests that I'm starting to form opinions about what I think they should look like.
+
+My most recent work has been JavaScript again. I've been writing Jest tests against one of our React apps. But
+rather than reaching for the testing patterns I'd become accustomed to over my years of JavaScripting, I'm finding
+that something's missing in my Jest tests! My experiences with RSpec have me longing for two features in Jest:
+
+<!-- more -->
+
+1. `let` blocks
+2. `context` blocks
+
+## 1. `let` blocks
```

Comment:
Minor suggestion, take it or leave it: you could update this header and the next one to be "What's a `let` block?" and "What's a `context` block?", just so that the header very clearly tells you what's coming in that section